### DISCUSSION TITLE
No demo/example usage of 'global' network

### DISCUSSION THREAD

**Discussion by mrlongzhang** [DeepRegNet/DeepReg#822] [Q&A]
Could anybody give me some hints on why no any examples on the global network? Is there any shortcoming of such method? The global and local network should come in pairs in my humble understanding. Waiting for any insights to light me up. Cheers

**Comment by YipengHu** [comment]
- it is just by chance that there isn't any global network alone examples - by the nature of most clinical applications being deformable. One can change the config file in most (if not all) tutorials to see if the globalnet works;
- there has been a "popular" idea that localnet or alike is not good for modelling large defromation. However, i personally do not see strong theoretical justification for this, and didn't see any evidence supporting this in our own applications. Therefore, i think this claim is at most application-dependent, so i do not see it should be paired with a globalnet necessarily.
my two cents of course :)
